Documentation/op-guide/grafana.json

```diff
@@ -115,17 +115,17 @@
                     "stack": false,
                     "steppedLine": false,
                     "targets": [{
-                            "expr": "sum(rate(grpc_server_started_total{grpc_type=\"unary\"} [1m]))",
+                            "expr": "sum(rate(grpc_server_started_total{grpc_type=\"unary\"}[1m])) by (grpc_method)",
```

Comment:
A typical scrape interval is 15s, 30s or even 60s, in such a case this range query would only rate over 4, 2 or 1 sample. I'd suggest to make this rather `5m`.